platform/ui/src/components/InvestigationalUseDialog/InvestigationalUseDialog.tsx

```diff
@@ -69,7 +69,7 @@ const InvestigationalUseDialog = ({
           <div className="flex flex-col">
             <div className="text-[19px] text-white">
               OHIF Viewer is{' '}
-              <span className="text-primary-light">for investigational use only</span>
+              <span className="text-primary-light">for investigational use only </span>
```

Comment:
This space shouldn't get added - that looks like a bug in the reformatter configuration.  Spaces do actually matter within some tags, so having it auto-add spaces isn't a good idea.